lib/guardian.rb

```diff
@@ -97,7 +97,7 @@ def in_any_groups?(group_ids)
   attr_reader :request
 
   def initialize(user = nil, request = nil)
-    @user = user.presence || AnonymousUser.new
+    @user = user.presence || Guardian::AnonymousUser.new
```

Comment:
This is just to make it more obvious which AnonymousUser class we are dealing with